cockatrice/src/gameselector.cpp

```diff
@@ -296,11 +295,13 @@ void GameSelector::actSelectedGameChanged(const QModelIndex &current, const QMod
     joinButton->setEnabled(game.player_count() < game.max_players() || overrideRestrictions);
 }
 
-void GameSelector::setFilteredGamesLabel()
+void GameSelector::updateTitle()
 {
     if (showFilters) {
         const int totalGames = gameListModel->rowCount();
         const int shownGames = totalGames - gameListProxyModel->getNumFilteredGames();
-        filteredGamesLabel->setText(tr("Games shown: %1 / %2").arg(shownGames).arg(totalGames));
```

Comment:
Doesn't this mean the filtered games label will never contain text and can thus be safely removed?

Reply:
Yup! Done.